docs/deploy.md

```diff
@@ -41,103 +35,150 @@ You should now see the built and tagged image in ``docker images``.
+...
 ```
 
 So in the case of the above line you would then run the following to complete a deploy to `staging`:
 
 ```sh
-$ inv staging deploy --tag=develop-bf47d7b
+(hip)$ inv staging deploy --tag=develop-bf47d7b
 ```
 
 
-## Local Deployment Environment Setup
-
+## Build and Deploy in one command
 
-### Access to Cluster
+To build and deploy in a single command:
 
+```sh
+(hip)$ inv staging image deploy
+```
 
-#### 1. Configure AWS named profile
+## Update or add a secret to the deployment
 
-Learning power uses the CaktusAccessRole to manage the aws resources for hip.  
-If you are unsure of what this means, contact Tech Support.  You should be able to find the credentials for that role in lastpass.
+If you need to add or change a secret in your environment, you'll first need to encrypt
+it and then add the encrypted value to your `deploy/host_vars/<environment>.yml` file,
+where `<environment>` represents the name of the environment you are updating, usually
+"staging" or "production.
 
-Once you have that setup, you should verify in your `~/.aws/credentials` file there is an entry for the main caktus IAM.
+1. Generate or obtain the new secret. Then encrypt it with this command:
 
-Link your hip profile to the Caktus Main credentials.
-  
-  1. Edit `~/.aws/credentials`
-  1. Either add or make sure you have the following filled out with the right credentials:
+	```sh
+	(hip)$ cd deploy
+	(hip)$ ansible-vault encrypt_string <secret>
+	!vault |
+	          $ANSIBLE_VAULT;1.1;AES256
+			  38646465363539613435316335663835373561346262383832303439623533376564636465666535
+			  3663396663373465633465636639636631303232343732380a343034376633323330386337653930
+			  36643366636334643839363763366335343266643431346435636264623634616538373863393534
+			  3937643532306231610a626333656461386433303335373361323330323466666130303063303863
+			  30623762363233643337653961633062346537643066663837633535336164623663
+    Encryption successful
+	```
+2. Copy everything from the `!vault` up until the `Encryption successful` message, but
+   not including the `Encryption successful` message.
 
+3. Paste it into `deploy/host_vars/<environment>.yml` as the value for the key that you
+   are adding or changing.
 
-```conf
-  [<CAKTUS_MAIN_PROFILE_NAME>]
-  aws_access_key_id = <LOOK-IN-LASTPASS>
-  aws_secret_access_key = <LOOK-IN-LASTPASS>
+## Running commands on a deployed enviroment
 
-  ...
+At a minimum, you may want to create a superuser for yourself on staging. Here is the
+command to do that.
 
-  [hip]
-  role_arn = arn:aws:iam::<10_DIGIT_hip_ACCOUNT_ID>:role/CaktusAccountAccessRole-Admins
-  source_profile = <CAKTUS_MAIN_PROFILE_NAME>
+```sh
+(hip)$ inv staging pod.shell
+appuser@app-84f486b849-pfp7k:/code$ python manage.py createsuperuser
+Email: me@example.com
+Password:
+Password (again):
+Superuser created successfully.
+appuser@app-84f486b849-pfp7k:/code$ exit
 ```
```

Comment:
It could be helpful to add a note about a `namespaces "hip-staging" not found` error here. Maybe something like "if you get a 'namespaces "hip-staging" not found' error, check out the "Configure access to the kubernetes cluster" section below to configure your local machine to have access to the cluster."